pkg/controller/logstorage/linseed/linseed_controller.go

```diff
@@ -205,6 +205,11 @@ func (r *LinseedSubController) Reconcile(ctx context.Context, request reconcile.
 		r.status.SetDegraded(operatorv1.ResourceReadError, "An error occurred while querying Tenant", err, reqLogger)
 		return reconcile.Result{}, err
 	}
+	err = validateTenant(tenant)
```

Comment:
I wonder if we should move this `validateTenant` inside the implementation of `GetTenant` - WDYT?

Reply:
Sure, but should we bunch up all errors under `operatorv1.ResourceReadError` ?

Reply:
I suppose we can leave it like so.